packages/core/src/constant/tx.ts

```diff
@@ -2,4 +2,4 @@ import BigNumber from "bignumber.js";
 
 export const TxGasCostTypeDefaultKey = "default_fee";
 export const TxDefaultGasDenom = "swth";
-export const TxDefaultGasCost = new BigNumber(1e7); // 0.1 SWTH
+export const DefaultGas = new BigNumber(1e8);
```

Comment:
are we setting the default to production default?
https://api.carbon.network/carbon/fee/v1/gas_costs/default_fee

Reply:
the TxDefaultGasCost is the default fee of 0.1SWTH, but we need to maintain the GAS (computational gas) as 10^8 because if not claiming of staking rewards and all will fail again. different things